SweetTweetApp/src/main/java/servletsForPosts/WorkWithJSON.java

```diff
@@ -0,0 +1,28 @@
+package servletsForPosts;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import net.dongliu.gson.GsonJava8TypeAdapterFactory;
+
+public class WorkWithJSON {
```

Comment:
Please learn another useful term, "Decorator". When we create an object, that wraps around some other object and adds some usefulties to it, this is called so. Therefore, this could be named like `GsonDecorator`.

The problem is,
you are again mixing up static and dynamic approaches. See, a statc field should never be initialized in an instance constructor,. Or else, if you or someone decides to create a second one instance of "WorkWithJson"m the state of `gson` field will be again destroyed - for all instances.
And again, static fields must be dealt with static methods, Per-instance methods must read/write only per-instance fields.

So rather make all the methods here static, and call the class just like "JsonUtil"